projects/cm_mcu/commands/BoardCommands.c

```diff
@@ -344,8 +346,14 @@ BaseType_t v38_ctl(int argc, char **argv, char *m)
   }
   UBaseType_t ffmask[2] = {0, 0};
   ffmask[whichFF - 1] = 0xe;
-  enable_3v8(ffmask, !turnOn);
-  snprintf(m, SCRATCH_SIZE, "%s: 3V8 turned %s for F%ld\r\n", argv[0],
+  int ret = enable_3v8(ffmask, !turnOn);
+  if (ret != 0) {
+    copied += snprintf(m + copied, SCRATCH_SIZE - copied, "enable 3v8 failed with %d\r\n", ret);
+    if (ret == 5)
+      snprintf(m + copied, SCRATCH_SIZE - copied, "please release semaphore \r\n");
+    return pdFALSE;
+  }
```

Comment:
do we need an `else` here?

Reply:
I think it is fine. This is analogous to a similar structure in other places such as the one in [ff_ctl](https://github.com/apollo-lhc/cm_mcu/blob/unify_acquireI2CSemaphore/projects/cm_mcu/commands/SensorControl.c#L917-L925)